Assets/Scripts/AI/AIManager.cs

```diff
@@ -1,8 +1,94 @@
 ﻿using UnityEngine;
 using System.Collections;
-
+/*
+stealth mini game triggered when one enemy is in the suspicious state
+one list of enemies
+*/
+ //public function that decrements timer of all ai in the alert/suspicious state
 public class AIManager : MonoBehaviour {
-	Transform[] AI_children;
+
+    public GameObject[] AiChildren;
+    public bool playerHidden;
+    public int numberChasing;
+    public static AIManager instace = null;
+
+    private static AIManager instance;
```

Comment:
Singleton instances have a different syntax for how they're instantiated and used. Usually the private variable is named _instance and the public one is called instance. If you're more curious about how it's normally used check the QK_Character_Movement instance declaration. I'll just merge these in and make the changes myself though.